Composer/packages/client/src/pages/home/Home.tsx

```diff
@@ -189,22 +189,19 @@ const Home: React.FC<RouteComponentProps> = () => {
                   src={noRecentBotsCover}
                 />
                 <div css={home.noRecentBotsDescription}>
-                  {formatMessage.rich(
-                    'Open the product tour to learn about Bot Framework Composer or <Link>create a new bot</Link>',
-                    {
-                      Link: ({ children }) => (
-                        <Link
-                          key="create-new-bot-link"
-                          onClick={() => {
-                            onClickNewBot();
-                            TelemetryClient.track('ToolbarButtonClicked', { name: 'new' });
-                          }}
-                        >
-                          {children}
-                        </Link>
-                      ),
-                    }
-                  )}
+                  {formatMessage.rich('<Link>Create a new bot to get started</Link>', {
```

Comment:
nit: Just looking at this should the link part just be for the "Create a new bot".